docs/examples.rst

```diff
@@ -15,29 +15,62 @@ should be addressed by the Ansible Creator Execution Environment project.
 
 .. _`Ansible Creator Execution Environment`: https://github.com/ansible/creator-ee
 
+Customizing the Docker Image Used by a Scenario/Platform
+========================================================
+
+The docker driver supports using pre-built images and ``docker build`` -ing
+local customizations for each scenario's platform. The Docker image used by a
+scenario is governed by the following configuration items:
+
+1. ``platforms[*].image``: Docker image name:tag to use as base image.
+2. ``platforms[*].pre_build_image``: Whether to customize base image or use
+   as-is[^1].
```

Comment:
Hey @timblaktu, thank you for this PR!

As this PR is already closed, I am not sure if this is the right place to mention this.
But let me do it anyway, if I should create a separate issue/PR, just let me know.

The footnotes in this PR do not render correctly in https://molecule.readthedocs.io/en/latest/examples.html.

The documentation is composed as [SphinxDoc/reStructuredText](https://www.sphinx-doc.org/en/master/) the footnote reference should look like [this](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#footnotes): `[#name]_`.

For example, this line could look like this:
```
as-is `[#dockerimplementation]_`.
```